CHANGELOG.md

```diff
@@ -4,21 +4,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+
+## Unreleased Sprint 69 Ending 2017-09-18
+
+### Added
+- Services
+  - Revert "Support TTL (Time To Live) value for services." [(#186)](https://github.com/ManageIQ/manageiq-content/pull/186)
```

Comment:
@JPrause This might be a little confusing.  In reality the logic was moved to a different method and this PR was reverting the old way.  So the logic was not removed (or reverted) from the current code, just moved in a previous PR.

Suggest we drop this one to avoid any confusion.

Reply:
Gotcha,...thanks, one update coming up.